.github/workflows/ci-windows.yml

```diff
@@ -0,0 +1,27 @@
+name: CI (windows)
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  run-ci:
+    name: Run
+
+    runs-on: windows-2019
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Add msbuild to PATH
+        uses: microsoft/setup-msbuild@v1
+
+      - name: Install dependencies
+        run: nuget restore Twingly.Search.sln
+
+      - name: Build project
+        run: msbuild Twingly.Search.sln /p:Configuration=Release
+
+      - name: Run tests
+        run: dotnet test ./Twingly.Search.Tests/bin/Release/Twingly.Search.Tests.dll
```

Comment:
I don't think this does what you hope it would do, check the output at https://github.com/twingly/twingly-search-api-dotnet/runs/1510183393?check_suite_focus=true#step:6:31 :)

Reply:
It sure didn't! It looks better now :)